private/rules/javadoc.bzl

```diff
@@ -65,12 +74,14 @@ def _javadoc_impl(ctx):
     # `None` https://github.com/bazelbuild/bazel/issues/10170). For this
     # reason we allow people to set javadocopts via the rule attrs.
 
+    javadocopts = ctx.attr.javadocopts if ctx.attr.javadocopts else _default_javadocopts
```

Comment:
Why not just set the default value of the attribute to be `_default_javadocopts`? Then you can continue to use `ctx.attr.javadocopts` throughout.

Reply:
That makes sense to me